components/tools/OmeroWeb/omeroweb/api/views.py

```diff
@@ -700,6 +700,15 @@ def _save_object(self, request, conn, object_json, **kwargs):
         # Therefore we ignore any details for now:
         obj.unloadDetails()
 
+        # Unlink children for Projects, Datasets and Screens to avoid
```

Comment:
Probably need a TODO so we can easily find it
and we need a card to review the graph round tripping since we will face it next  round with image and roi saving for example